src/wp-includes/block-editor.php

```diff
@@ -285,10 +285,14 @@ function get_block_editor_settings( array $custom_settings, $block_editor_contex
 	$theme_json = WP_Theme_JSON_Resolver::get_merged_data( $editor_settings );
 
 	if ( WP_Theme_JSON_Resolver::theme_has_support() ) {
-		$editor_settings['styles'][] = array( 'css' => $theme_json->get_stylesheet( 'block_styles' ) );
+		$editor_settings['styles'][] = array(
+			'css'            => $theme_json->get_stylesheet( 'block_styles' ),
+			'__unstableType' => 'globalStyles',
+		);
 		$editor_settings['styles'][] = array(
 			'css'                     => $theme_json->get_stylesheet( 'css_variables' ),
 			'__experimentalNoWrapper' => true,
+			'__unstableType'          => 'globalStyles',
```

Comment:
"Global styles" has several origins it could be core, theme, and user. Do you think it would make sense/would be useful to also have origin indicators for global styles? It would be complex given how our theme.json merge works.

Reply:
I considered it, but the concept of "origin" doesn't match well this use case: `globalStyles` is the result of merging three different sources of data.